Userland/DevTools/UserspaceEmulator/main.cpp

```diff
@@ -43,15 +44,14 @@ int main(int argc, char** argv, char** env)
     if (dump_profile && profile_instruction_interval == 0)
         profile_instruction_interval = 128;
 
-    DeprecatedString executable_path;
-    if (arguments[0].contains("/"sv))
-        executable_path = Core::DeprecatedFile::real_path_for(arguments[0]);
-    else
-        executable_path = Core::DeprecatedFile::resolve_executable_from_environment(arguments[0]).value_or({});
-    if (executable_path.is_empty()) {
+    auto executable_path_or_error = arguments[0].contains('/')
+        ? FileSystem::real_path(arguments[0])
```

Comment:
`resolve_executable_from_environment` already checks if the path contains a `/`, maybe we should make it translate the path into an absolute one instead? It certainly looks like something that other callers of that function might want as well. Having a single non-resolving branch in a resolve function is certainly weird.

Reply:
I really want to modify UserspaceEmulator as little as possible, because it doesn't currently compile and I don't want to conflict with PRs that might bring it back to life.

Whoever fixes UserspaceEmulator can definitely cange this invocation however they see fit, but I don't think it should be changed in this PR.